lib/clientv2/ippool.go

```diff
@@ -145,6 +193,41 @@ func (_ ipPools) validateAndSetDefaults(pool *apiv2.IPPool) error {
 		}
 	}
 
+	// Normalize the CIDR before persisting.
+	pool.Spec.CIDR = cidr.String()
+
+	// If there was a previous CIDR, validate the CIDR has not changed.  Since we
+	// are using noramlized CIDRs we can just do a simple string comparison.
+	if previousCIDR != "" && previousCIDR != pool.Spec.CIDR {
```

Comment:
This is essentially saying "Only perform this validation on Update", right?

Reply:
Yup.